tests/conftest.py

```diff
@@ -46,12 +46,13 @@ def mock_result_link_response():
     return {"some": "result"}
 
 
-def mock_response(request, context):
+def mock_response(request, context):  # noqa: ARG001
```

Comment:
I am wondering if doing this helps us avoid  `# noqa: ARG001`

```suggestion
def mock_response(request, _):  
```